File: CampanhaKg.Domain/models/Fraternity.cs
using System.Collections.Generic;
using CampanhaKg.Domain.Identity;

namespace CampanhaKg.Domain.models
{
    public class Fraternity : Address
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public List<Campaign> Campaigns { get; set; }
        public int UserId { get; set; }
        public User user { get; set; }
        public string photo { get; set; }



    }
}

File: CampanhaKg-App/src/app/_models/Fraternity.ts
import { Campaign } from './Campaign';
import { Address } from './Address';
import { User } from './User';

export interface Fraternity {
    id: number;
    name: string;
    rua: string;
    numero: number;
    complemento: string;
    bairro: string;
    cidade: string;
    uf: string;
    zipcode: string;
    userId: number;
    user: User;





}
